cupy/linalg/norms.py

```diff
@@ -97,7 +103,11 @@ def norm(x, ord=None, axis=None, keepdims=False):
                              (axis, x.shape))
         if row_axis == col_axis:
             raise ValueError('Duplicate axes given.')
-        if ord == 1:
+        if ord == 2:
+            ret = _multi_svd_norm(x, row_axis, col_axis, cupy.amax)
+        elif ord == -2:
+            ret = _multi_svd_norm(x, row_axis, col_axis, cupy.amin)
```

Comment:
SVD returns the singular values in descending order.  `functools.partial(cupy.take, indices=0)` or `functools.partial(cupy.take, indices=-1)` could be used, while SVD is much heavier than max/min.

Reply:
I have added that.